pilot/prompts/development/task/breakdown.prompt

```diff
@@ -35,4 +35,7 @@ You do not need to make any automated tests work.
 {% endif %}
 DO NOT specify commands to create any folders or files, they will be created automatically - just specify the relative path to each file that needs to be written.
 
+**IMPORTANT**
+You must not tell me to run a command in the database or anything OS related - only if some dependencies need to be installed. If there is a need to run an OS related command, specifically tell me that this should be labeled as "Human Intervention" and explain what does the human need to do.
```

Comment:
same comment as above for same prompt. Also, it might be good to have it in `components` folder so in future there is only 1 place where we have to make changes.